source/docs/index.md

```diff
@@ -68,24 +68,28 @@ Once all the requirements are installed, you can install Hexo with npm:
 $ npm install -g hexo-cli
 ```
 
-### Advanced installation and usage
-
-Advanced users may prefer to install and use `hexo` package instead.
+The Hexo installed using the commands above is globally effective. If the installation is successful, executing the following command will display the version information.
 
 ``` bash
-$ npm install hexo
+$ hexo --version
 ```
 
-Once installed, you can run Hexo in two ways:
+{% note info %}
+For advanced users familiar with npm, you can opt for a local installation of the `hexo` package instead of a global one. This approach can help avoid potential permission issues associated with global installations. For example, you can clone an existing Hexo blog repository, and after installing dependencies, execute Hexo commands using `npx hexo`.
 
-1. `npx hexo <command>`
-2. Linux users can set relative path of `node_modules/` folder:
+``` bash
+$ git clone https://github.com/hexojs/hexo-starter
+$ cd hexo-starter
+$ npm install
+$ npx hexo --version
+```
 
-  ``` bash
-  echo 'PATH="$PATH:./node_modules/.bin"' >> ~/.profile
-  ```
+If you choose a local installation of `hexo`, Linux users can add the `node_modules` directory in the Hexo directory to their PATH environment variable to directly use `hexo <command>`:
 
-  then run Hexo using `hexo <command>`
+```bash
+echo 'PATH="$PATH:./node_modules/.bin"' >> ~/.profile
```

Comment:
I suggest to delete this part of modifying `path` setting. Modifying `path` may be a dangerous action. For example, some `bin` of `node_modules` overwrite some user's software, or some `node_modules` contain dangerous actions or even malicious code. For those familiar with Linux, they know how to do. For ordinary users, just recommend that they use `npx hexo`. 

If anyone thinks `npx hexo` is too long, consider using `alias`